motr/client_init.c

```diff
@@ -1222,6 +1238,8 @@ static int initlift_layouts(struct m0_sm *mach)
 	return M0_RC(initlift_get_next_floor(m0c));
 }
 
+#define MAX_CLIENT_INIT_RETRIES 1000
+int retry_count = 0;
```

Comment:
Use static here.

Reply:
Move these into the function itself: to minimize the scope of variables and define.

Reply:
Done. Made retry_count static